src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -93,6 +95,15 @@ def image_and_background(data, meta, log, m):
     xmin, xmax = data.flux.x.min().values, data.flux.x.max().values
     ymin, ymax = data.flux.y.min().values, data.flux.y.max().values
 
+    # FINDME: Max calculation is broken and I can't figure out how to fix it
+    # vmax = np.nanmax(~data.mask.values*subdata)/10
```

Comment:
This was my attempt to fix it, but it still doesn't work.  The previous version was returning NaN, which messed up the figure scaling.

Reply:
Can this be done with masked arrays?

Reply:
Ok, I got the calculation to work with `masked_invalid()` but it's still super sensitive to unmasked hot pixels.  I decided to leave it commented out in hopes of improving it later.  Fortunately, since the NIR BG is very predictable, I don't see a need to change the current implementation until we investigate some MIRI data.